public/app/core/services/keybindingSrv.ts

```diff
@@ -41,10 +41,6 @@ export class KeybindingSrv {
       this.bind('g a', this.openAlerting);
       this.bind('g p', this.goToProfile);
       this.bind('g e', this.goToExplore);
-      if (!config.featureToggles.topNavCommandPalette) {
```

Comment:
wonder whether this should be gated on `topnav` instead of `topNavCommandPalette`? 🤔 this means if we turn `topnav` off, there is now no keyboard shortcut to open search

Reply:
good shout. I was actually meaning to ask here - what if (with topnav) we change these shortcuts to instead open command palette, to try to avoid breaking habits.

Reply:
idk, my gut is we should just standardise on the one tbh. discussed it a bit [here](https://github.com/grafana/grafana/pull/62286). if we want to support `f` or `s o`, that's what we should show in the input and we should have `cmd+k` as the undocumented shortcut since it's a common key combination. either way we definitely shouldn't have 3 😬